src/ui/actions/timeline.ts

```diff
@@ -264,6 +269,44 @@ export function seek(
   };
 }
 
+export function jumpToReduxCode(
+  point: ExecutionPoint,
+  time: number,
+  sourcesState: SourcesState
+): UIThunkAction {
+  return async (dispatch, _, { replayClient }) => {
+    dispatch(framePositionsCleared());
+
+    const pauseId = await pauseIdCache.readAsync(replayClient, point, time);
+    const frames = (await getPauseFramesAsync(replayClient, pauseId, sourcesState)) ?? [];
+    const filteredPauseFrames = frames.filter(frame => {
+      const { source } = frame;
+      if (!source) {
+        return false;
+      }
+
+      // Filter out everything in `node_modules`, so we have just app code left
+      // TODO There may be times when we care about renders queued by lib code
+      // TODO See about just filtering out React instead?
+      return !MORE_IGNORABLE_PARTIAL_URLS.some(partialUrl => source.url?.includes(partialUrl));
+    });
+
+    const frameSteps = await frameStepsCache.readAsync(
+      replayClient,
+      pauseId,
+      // The first 2 elements in filtered pause frames are from replay's redux stub, so they should be ignored
+      // The 3rd element is the user function that calls it, and will most likely be the `dispatch` call
+      filteredPauseFrames[2].protocolId
+    );
+
+    if (frameSteps) {
+      const frameStep = frameSteps[0];
```

Comment:
:hand: It's probably going to be a bit trickier than this. The actual `dispatch()` call ought to be _somewhere_ in this frame, but there's no guarantee it's the _first_ step in the frame. 

I think the `ReactPanel.tsx` code has some logic that tries to find the right framestep for the line of code that led into this call.

Reply:
Oh interesting, @jasonLaster mentioned that it should be the first one but yeah `ReactPanel.tsx` does have some logic for finding the right one

Reply:
ooh interesting.